mangaki/mangaki/templates/base.html

```diff
@@ -59,35 +61,45 @@
                 <div class="collapse navbar-collapse" id="menu-collapse">
                     <ul class="nav navbar-nav">
                         <li class="dropdown">
-                            <a href="{% url 'about' %}">
+                            <a href="{% url 'about' 'fr' %}">
                                 C'est quoi ?
                             </a>
                         </li>
+                        <li class="dropdown">
```

Comment:
In fact, I think we are doing it wrong.

We should rely on `Accept-Language` header to show the good `What is it` rather than showing them *all*.